sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetRowConverter.scala

```diff
@@ -178,8 +179,17 @@ private[parquet] class ParquetRowConverter(
 
   // Converters for each field.
   private[this] val fieldConverters: Array[Converter with HasParentContainerUpdater] = {
```

Comment:
@kimtkyeom . @viirya means the following in line 182.
```scala
- val catalystFieldNameToIndex = if (caseSensitive) {
+ val catalystFieldNameToIndex = if (SQLConf.get.caseSensitiveAnalysis) {
```

Reply:
Aha, I will change it. Thank you for comment.

Reply:
Oh, wait a minute for this.

Reply:
@viirya . Is the above what you mean?

Reply:
Yea, thanks @dongjoon-hyun.